nextjs-app/app/page.tsx

```diff
@@ -1,5 +1,6 @@
 import Link from "next/link";
 
+//eslint-disable-next-line
```

Comment:
I think we can disable this rule globally if we will use a lot of server component in the future

Reply:
No problem, that makes sense to me!